src/main/java/teammates/ui/controller/InstructorFeedbackEditSaveAction.java

```diff
@@ -30,14 +26,15 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                 Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION);
 
         InstructorFeedbackEditPageData data = new InstructorFeedbackEditPageData(account, sessionToken);
-        FeedbackSessionAttributes feedbackSession = extractFeedbackSessionData(false);
 
-        // A session opening reminder email is always sent as students
-        // without accounts need to receive the email to be able to respond
-        feedbackSession.setOpeningEmailEnabled(true);
+        // This is only for validation to pass; it will be overridden with its existing value at the logic layer
+        String dummyCreatorEmail = "dummy@example.com";
```

Comment:
I think it's better to not use a dummy value. Can extract the instructor from line 23 and use `instructor.email`

Reply:
The instructor may not be the creator (e.g. a co-owner).

Reply:
I see. Nvm then.